### ISSUE TITLE
Play From Here in the test menu

### ISSUE THREAD

**Issue by mattmundell** [SeleniumHQ/selenium-ide#1247]
## 🚀 Feature Proposal

Add a menu item to the per-test menu, that plays the test and all following tests.

## Motivation

I have a bunch of tests.  Often one of them fails and I stop the process to sort it out.  Then I want to carry on from there, but the only options seem to be to run them all, or to run the rest manually one by one.

## Example
1. Open a project with 10 tests.
2. Click "Run all tests".
2. Click the stop button during test number 5.
3. Right click on test number 5 in the tests sidebar.
4. Click **new menu item** "Play from here".
5. Tests number 5-10 are run.

**Comment by toddtarsi** [comment]
A couple semantic critiques:

1. This is expected of the stop button. Perhaps this should occur with the pause button, but the stop button dissolves playback state by design.
2. This will most likely not happen in v3. This will be something I keep in mind in v4. It seems like this is really asking for a more configurable and active playback test list state.

**Comment by mattmundell** [comment]
Yes, being able to select all unrun/failed/non-pass tests would solve it.

Pause might be a problem, for example i could run a single test lower down and then i would not be able to continue from the failed test.
